recipes/spint/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "spint" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 64beb8bf3daaf299d6b0ca6051d2c2213129c09e62fd0a0e6f36374a88b8e9c8
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python>=3
+    - pip
+  run:
+    - python>=3
+    - scipy
+    - numpy
+    - libpysal
```

Comment:
![image](https://user-images.githubusercontent.com/7359284/61813012-93d23680-adf9-11e9-9580-1e5b9a70a932.png)
Seems to have some issues related to the dependency versions (according to the error message, some specifications are not compatible)? Maybe trying restricting the versions of the dependencies (especially those pysal submodules)? Sth similar to the giddy conda-forge recipe:
![image](https://user-images.githubusercontent.com/7359284/61813345-55894700-adfa-11e9-9080-ec97ae9e401a.png)

Reply:
Just tried that and now seems all OS fail.

Reply:
Because the most recent version of spreg on conda-forge and pypi is 1.0.4 https://github.com/conda-forge/spreg-feedstock/blob/master/recipe/meta.yaml
Just opened an issue on spreg https://github.com/pysal/spreg/issues/26

Reply:
Okay, switch to the older version to match pypi and am giving a shot

Reply:
Hmmm that did't seem to solve it.

Reply:
Seems all the failures point to `Package vc conflicts for:`. Probably have sth to do with   `build: - {{ compiler('c') }}`?

Reply:
Not sure why you added the c compiler here? Can you remove it and try again?

Reply:
I think it was a default option. Removing it works! Thanks!